### ISSUE TITLE
Which mask token is used in T5 training?

### ISSUE THREAD

**Issue by StephanHeijl** [agemagician/ProtTrans#42]
Hi ProtTrans team, I am working with the ProtT5 model, and busy finetuning it on other sequences. The paper states that MLM is used similar to the other models, with `[MASK]` tokens being introduced at random positions. It seems however that `[MASK]` is not a valid item in the T5 tokenizer vocabular:

```
{'<pad>': 0,
 '</s>': 1,
 '<unk>': 2,
 '▁A': 3,
 '▁L': 4,
 '▁G': 5,
 '▁V': 6,
 '▁S': 7,
 '▁R': 8,
 '▁E': 9,
 '▁D': 10,
 '▁T': 11,
 '▁I': 12,
 '▁P': 13,
 '▁K': 14,
 '▁F': 15,
 '▁Q': 16,
 '▁N': 17,
 '▁Y': 18,
 '▁M': 19,
 '▁H': 20,
 '▁W': 21,
 '▁C': 22,
 '▁X': 23,
 '▁B': 24,
 '▁O': 25,
 '▁U': 26,
 '▁Z': 27,
 '<extra_id_99>': 28,
 '<extra_id_98>': 29,
[ommitted for space]
 '<extra_id_1>': 126,
 '<extra_id_0>': 127}
```
I tried using `<extra_id_0>` as suggested in the T5 paper, but this yielded very low accuracies in initial finetuning. Was another specific token used, or do you actually insert the `[MASK]` string, which translates to id 2?

I also wanted to thank you sincerely for your amazing world and your decision to release ProtTrans pretraine models to the public!

**Comment by StephanHeijl** [comment]
I should have searched the closed issues, my apologies. This is answered in #39 .
